config/action.d/shorewall-ipset-proto6.conf

```diff
@@ -0,0 +1,84 @@
+# Fail2Ban configuration file
+#
+# Author: Eduardo Diaz
+#
+# This is for ipset protocol 6 (and hopefully later) (ipset v6.14).
+# for shorewall
+#
+# Use this setting in jail.conf to modify the name and the max time in every jain
+# action   = shorewall-ipset-proto6[name=SSH, bantime=10000]
```

Comment:
you must use a name and a bantime to pass this parameters to the aplication you create two ipset one for every jail